### DISCUSSION TITLE
Redirecting a domain with a port

### DISCUSSION THREAD

**Discussion by realSZ27** [NginxProxyManager/nginx-proxy-manager#2941] [Q&A] [answered]
On my home server, I moved all services to NPM and would like to redirect an old domain like **http://example.com:6054** to a new one like **https://example.com**. Would this be possible at all? I'm sort of new to this kind of thing.

**Comment by slaygirlz** [comment, marked as answer]
late but yes

**Comment by realSZ27** [comment]
I figured it out🤣. Not sure when I made this, why I didn't just ... *try it*
